tools/explorer/lib/ontology_wrapper.py

```diff
@@ -1,14 +1,17 @@
-"""Ontology class for ontology explorer."""
+"""
+Ontology wrapper class for ontology explorer exposing certain functionality
+of the Digital Buildings Ontology.
+"""
 from typing import List
 
-from model import EntityTypeField as ETF
-from model import StandardField
-
 from yamlformat.validator.entity_type_lib import EntityType
+from yamlformat.validator.entity_type_manager import EntityTypeManager
 from yamlformat.validator.presubmit_validate_types_lib import ConfigUniverse
 
+from lib.model import StandardField
+from lib.model import EntityTypeField as ETF
```

Comment:
nit: the alias here doesn't seem necessary and actually makes it more confusing imho

Reply:
I did this purely to shorten the name so that I could fit the linter's line character limit in a few places. I may be able to fix.